Cargo.lock

```diff
@@ -1065,7 +1065,7 @@ dependencies = [
 [[package]]
 name = "dplane-rpc"
 version = "1.0.2"
-source = "git+https://github.com/githedgehog/dplane-rpc.git#019206ff443645684596fbef49470163f5ae58d6"
+source = "git+https://github.com/githedgehog/dplane-rpc.git#79a7929186716c3f463866614a9b23ea4b641949"
```

Comment:
@sergeymatov Can you please review?

Reply:
Sorry this is `dplane-rpc` :facepalm:, probably for @Fredi-raspall to take a look instead.

Reply:
The only difference between the two SHAs is this commit:

https://github.com/githedgehog/dplane-rpc/commit/79a7929186716c3f463866614a9b23ea4b641949

Looks like we're rather on the safe side for merging.